src/fb-base-blocks/navigation/navigation.jsx

```diff
@@ -6,7 +7,7 @@ const Navigation = (props) => {
     titleTag,
     children,
     mods = {},
-    dropdown = { mods: {}, handle: { content: 'Menu' } },
+    dropdown = { mods: {}, handle: { content: t`Menu` } },
```

Comment:
I'm a bit oldschool-ish when I'm dealing with i18n, and prefer explicit enums rather than tag-literal-something.

So, here's an important question — will this tool notify us if there's a string that is not completely translated? By “notify” I mean “panic, scream, crash building process, call the cops”.

Reply:
At first, let me divide the definition of "string is not translated" in two cases:

1) I've added some visible (UI-related) text with no additional transformation. Example: `<p>Something new</p>`.
2) I've added some text again and have applied transform function from `lingui` to it using `Trans` or `t`. Example: `<p><Trans>Something new</Trans></p>`

In first case, no one will ever know that this text is supposed to be translated. It's just a text. This is a developers responsibility zone. No panic, no cops.

In second case, we explicitly say that this text MUST be translated (because of <Trans> or <t>). To appear in resulting translated form the text should be extracted (`npm run extract`) and compiled (`npm run compile`).

This is where `compile --strict` (https://lingui.js.org/ref/cli.html#compile) comes to the rescue. In the `test` script, CI runs `extract` and `compile --strict`. If there are some strings with no translation, test script will fail (because of non-zero missing translations).

So yes, in case of explicit demand of translation there will be panic and crashed test process.

Reply:
Alright 🤔 

I'm looking at npm scripts and see `compile` script there, but don't see anything that runs this script. So, I guess it should be run manually every time we change locales? I mean, there's no run of this script on prepublish hook or something 🤔 

In this case, should we add a note into README about it?

https://github.com/funbox/blueprinter#development

Reply:
`compile` also can be run with `--watch`, so maybe we should add it into `dev` script. Or maybe we shouldn't, I'm not sure 🤷

Reply:
Updated readme with i18n note: https://github.com/funbox/blueprinter/pull/10/commits/feaf92605d9497d597c296c4f98f540a71c74a07

`compile --watch` watches changes in JSON catalogs which are also should be in sync with source files. So, JSON files also should be watched.

I thought it would be easier to add all new texts in the source code in English (for this is the source locale) and in the end of work just run `extract` and `compile` once to supply texts with translation.